include/llvm/Bitcode/RecordLayout.h

```diff
@@ -97,6 +97,12 @@ class BCFixed : public impl::BCField<> {
     abbrev.Add(llvm::BitCodeAbbrevOp(llvm::BitCodeAbbrevOp::Fixed, Width));
   }
 
+  static void assertValid(const bool &data) {
+    // MSVC warns: '>=' unsafe use of type 'bool' in operation.
```

Comment:
The warning seems reasonable.  I'm surprised that clang doesn't warn on that (perhaps with -pedantic).

I would just drop the comment.